api/session/data.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+
+import requests
+from api.session.models.account import Account
+from config.env import env
+
+
+# Account Store Endpoints
+ACCOUNT_ENDPOINT = "/accounts?email_address={email}"
```

Comment:
🔧 I think that anything which is configurable should be in configuration.

Reply:
Good shout - moved